docs/start/install-qiskit-source.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Install Qiskit from source
+description: Install Qiskit from source
+
+---
+# Install Qiskit from source
+
+Installing Qiskit from source allows you to access the current development version, instead of using the version in the Python Package Index (PyPI) repository. This lets you inspect and extend the latest version of the Qiskit code more efficiently. 
+
+## Create and activate a new virtual environment
+
+First, create a new virtual environment for Qiskit. 
+
+```python
+  python3 -m venv QiskitDevenv
+  source QiskitDevenv/bin/activate
+```
+
+## Install the Rust compiler
+
+The Rust compiler must be installed in your new virtual environment. To install the Rust compiler, use the cross-platform Rust installer [rustup](https://rustup.rs/) or [another install method.](https://forge.rust-lang.org/infra/other-installation-methods.html) 
+
+## Install Qiskit
+
+Follow these steps to install Qiskit:
+
+1. Clone the Qiskit repository.
+
+```python
+  git clone https://github.com/Qiskit/qiskit-terra.git
+```
+
+2. Change to the `qiskit-terra` directory. 
+
+```python
+  cd qiskit-terra
+```
+
+3. (Optional) If you want to run tests or linting checks, install the developer requirements.
+
+```python
+  pip install -r requirements-dev.txt
+```
+
+4. Install `qiskit-terra`. 
+
+  * **Standard install**: 
+
+    ```python
+      pip install .
+    ```
+
+  * **Editable mode**: In this mode, you don't need to reinstall Qiskit when there are code changes to the project.
+
+    ```python
+      pip install -e .
+    ```
+
+    In editable mode, the compiled extensions are built in _debug mode_ without optimizations. This affects the runtime performance of the compiled code. To build the compiled extensions with optimizations enabled, run the following command to rebuild the binary in _release mode_:
+
+    ```python
+      python setup.py build_rust --release --inplace
+    ```
+
+    <Admonition type="note">
+    If you are working on Rust code in Qiskit, you need to rebuild the extension code every time you make a local change. In editable mode, the Rust extension is only built when the install command is run, so local changes you make to the Rust code are not reflected in the installed package unless you rebuild the extension by rerunning `build_rust` (with or without `--release`, depending on whether you want to build in release or debug mode).
+    </Admonition>
+
+## Run example code
+
+You can run Qiskit code examples by running the following command:
+
+```python
+  python examples/python/using_qiskit_terra_level_0.py
+```
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
```

Comment:
I think people who come to this page might be more interested in trying out the development version of qiskit or possibly contributing. Maybe these are more appropriate next steps:

- Want to contribute to the open-source Qiskit SDK? read the contributing guidelines: https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md

Reply:
I think linking to the contributing information is a good idea, since "extending Qiskit" is given in the motivation summary at the top of the page (very correctly!) as a reason for wanting to do this.